microprofile/cors/src/test/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2019, 2020 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+cors.paths.0.path-prefix=/cors3
```

Comment:
In MP - shouldn't we map this to resource class and resource method?
Maybe the approach of fully qualified classname + method name + CORS parameters would be easier for MP users?
Such as:
```
io.helidon.microprofile.cors.CrossOriginTest.CorsResource2.options.value=http://foo.bar,http://bar.foo
io.helidon.microprofile.cors.CrossOriginTest.CorsResource2.options.allowHeaders=X-foo,X-bar
io.helidon.microprofile.cors.CrossOriginTest.CorsResource2.options.allowMethods=DELETE,PUT
io.helidon.microprofile.cors.CrossOriginTest.CorsResource2.options.allowCredentials=true
io.helidon.microprofile.cors.CrossOriginTest.CorsResource2.options.maxAge=-1
```

Reply:
The existing format has several advantages:
-  Much less verbose.
-  Immune to refactoring of the code.
-  Supports use cases that the alternative does not. (see below)
- Matches more closely to the mental model for the person setting up the configuration.

The existing format allows _path expressions_ for the path. This lets users easily configure multiple related endpoints in one CORS setting, including using a single entry for the entire app (which seems to be what's often done in the wild).

Also, the alternative described here does not allow for one resource to have multiple endpoints at different paths with the same HTTP method. To support that the alternative format would have to also include either the Java method name or the URI path. In that case why not just use path expressions as in the existing format?

Paths are likely to be more stable than implementation packages and class names because paths are used externally; code is subject to refactoring.